bundle/phases/deploy.go

```diff
@@ -170,11 +176,6 @@ func deployPrepare(ctx context.Context, b *bundle.Bundle) {
 		// SwitchToPatchedWheels must be run after ExpandGlobReferences and after build phase because it Artifact.Source and Artifact.Patched populated
 		libraries.SwitchToPatchedWheels(),
 
-		// libraries.Upload() not just uploads but also replaces local paths with remote paths.
-		// TransformWheelTask depends on it and planning also depends on it.
-		libraries.Upload(),
-		trampoline.TransformWheelTask(),
-
 		mutator.ResolveVariableReferencesOnlyResources(
```

Comment:
We now call `ResolveVariableReferencesOnlyResources` before libraries upload, which appears to be okay because `ResolveVariableReferencesOnlyResources` does not depend on libraries.

Reply:
Ah, yes, you're right